bin/terser

```diff
@@ -101,6 +102,10 @@ if (program.comments) {
     if (typeof options.output != "object") options.output = {};
     options.output.comments = typeof program.comments == "string" ? program.comments : "some";
 }
+if (program.preserveAnnotations) {
```

Comment:
This option is not needed. If someone uses -b preserve_annotations=true they can set the option. The fact that this is confusing because of the beautify option is a separate problem.

Reply:
thank you!